### DISCUSSION TITLE
隧道被封禁，删不了也编辑不了，求助

### DISCUSSION THREAD

**Discussion by soulxyz** [natfrp/wiki#266] [使用问题]
好早好早之前被封禁的，Http隧道，应该是里面有一个没有备案的域名，当时免费用户隧道是5个，也就没怎么管了，后来免费用户下调到2个，现在想删掉这个隧道也删不了……尬住了，这怎么处理
隧道ID 307812

**Comment by soulxyz** [comment]
错误 [403]: 隧道 #307812 已被管理员封禁, 无法删除

**Reply by sunshine8533** [reply]
同咨询，我也遇到了这个问题

**Reply by sunshine8533** [reply]
还是不太明白，为什么封禁我的隧道，也没有给出来理由。
需要备案什么域名呢？

[a comment by an automated account is not shown]
